magento/adobe-stock-integration
# #issue-904 Added view details slide panel for an image in Media Gallery

### Description (*)
This PR adds an ability to view image details in Media Gallery.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#904 View details slide panel for an image in media gallery

### Manual testing scenarios (*)

1. Login to Admin panel
2. Open Media Gallery (i.e. Catalog -> Category -> expand Content -> Select from Gallery button)
3. Click on three dots near to image that you want to remove
4. Click on the "View Details" option
<img width="1440" alt="Screenshot 2020-03-20 at 14 52 33" src="https://user-images.githubusercontent.com/31502344/77165812-b79a6900-6abb-11ea-9670-6208e427c972.png">
<img width="1440" alt="Screenshot 2020-03-20 at 14 52 47" src="https://user-images.githubusercontent.com/31502344/77165818-bc5f1d00-6abb-11ea-8d36-ca53c1b2db0b.png">
<img width="1440" alt="Screenshot 2020-03-20 at 14 52 58" src="https://user-images.githubusercontent.com/31502344/77165823-bec17700-6abb-11ea-8c7e-0df344306708.png">

As a result, you should see the slide panel with image details.
